src/core/payment/SubstituteAndExcludePayToAddressesForSuperCharge.ts

```diff
@@ -19,10 +19,12 @@ export class SubstituteAndExcludePayToAddressesForSuperCharge implements ISubsti
         //TODO: Find a different way to handle this without doing require in const
         const path = require('path');
 
+        const knownSubstitutePayToFile = path.join('src', 'data', '.knownSubstitutePayTo');
```

Comment:
I may not follow things correctly, but it looked like you were going to pre-load this in the configuration, but the file name is also hard coded here? Is the intention to use the configuration manager's getCommissionRates?

Reply:
This has been changed in the last commit.
There will be 4 files included:

- knownNegotiatedFees this will contain the list of wallets where we know already that the rate is 0.08 and the supercharged wallets will be burned
- knownSubstitutePayTo: this will contain all the known delegating wallets + associated return wallets. This is to avoid the operational challenges MF was facing related to several BPs not sending the rewards to the right address.

Then the BPs will always have the flexibility and ability to keep the .negotiatedFees and .substitutePayTo to define their own pool commission rates and return wallets.